daffodil-lib/src/main/scala/org/apache/daffodil/cookers/EntityReplacer.scala

```diff
@@ -810,7 +809,7 @@ class DelimiterCookerNoES(pn: String) extends ListOfString1OrMoreLiteral(pn, tru
   override val oneLiteralCooker: StringLiteralBase =
     new StringLiteralNoCharClassEntities(propName, true) {
 
-      override protected def noCharClassEntities(raw: String, context: ThrowsSDE): Unit = {
+      override protected def noCharClassEntities(raw: String, disallowedCharClassEntities: String, context: ThrowsSDE): Unit = {
         // TODO: this isn't quite right, as it will allow combined delimiters
```

Comment:
I added the parameter disallowedCharClassEntities here to match the def above, but I am not using the function because of the additional error message test for these items. Perhaps there is a better way to handle this?